### PR TITLE
Add release scripts

### PR DESCRIPTION
**Pull request by rcurtin**
This just adds the two release scripts used to release a version of mlpack automatically, just like in ensmallen.

Anyone should be able to run `scripts/release-mlpack.sh <major> <minor> <patch>` in order to automatically open a pull request for a release, and then mlpack-bot should do the rest automatically once that automatically-opened PR is merged.  I'm excited about this!  It can help remove me as a bottleneck to releases. :)

### CODE REVIEW

File: scripts/release-mlpack.sh [hunk 1]
```diff
@@ -0,0 +1,221 @@
+#!/bin/bash
```
Comment (zoq) [inline scripts/release-mlpack.sh:1]:
```suggestion
#!/usr/bin/env bash
```

File: scripts/update-website-after-release.sh [hunk 2]
```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
```
Comment (zoq) [inline scripts/update-website-after-release.sh:1]:
```suggestion
#!/usr/bin/env bash
```

File: scripts/release-mlpack.sh (hunk 1 above)
Comment (rcurtin) [reply scripts/release-mlpack.sh:1]: Hehe, good point :)

**Comment by shrit** [comment]
Great, now it will be easier and faster to make releases.
Many thanks, @rcurtin

**Review by zoq — APPROVED** [review]
This looks good to me, thanks for putting this together.

**Pull request merged**